### PR TITLE
Integrate prettier along with eslint

### PR DESCRIPTION
**Pull request by gdostie**
This is awesome, it works very nicely :hyped:
Can't wait to have this in our other repos, makes [coveo/tsjs](https://github.com/coveo/tsjs) pretty useless :crying_cat_face:

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,33 +28,41 @@ const Colors = {
  * @param {string} [config.moduleName = ''] - The name of the module to export
  * @param {RegExp[]} [config.internalImportPaths = []] - The internal import paths that will be parsed out
  */
-module.exports = function (config) {
-    return through.obj(function (vinylFile, encoding, callback) {
+module.exports = function(config) {
```
Comment (GermainBergeron) [inline index.js]: Can you modify prettier to leave the space after the function?

Comment (GermainBergeron) [reply index.js]: Why don't you use arrow functions?

File: index.js [hunk 2]
```diff
@@ -76,7 +84,7 @@ function parseImportStatement(regExpArgs, config) {
     return '';
 }
 
-function isInternalImport(path, {internalImportPaths = []}) {
+function isInternalImport(path, { internalImportPaths = [] }) {
```
Comment (GermainBergeron) [inline index.js]: is it possible to configure this?

**Review by GermainBergeron — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (gdostie) [reply index.js]: I can't leave a space after the function, Prettier is pretty opinionated. Those are the only options you can play with: https://prettier.io/docs/en/options.html

I'll switch for arrow functions.

File: index.js (hunk 2 above)
Comment (gdostie) [reply index.js]: This one yes https://prettier.io/docs/en/options.html#bracket-spacing

**Pull request merged**